bag_rdr.cpp

```diff
@@ -1035,6 +1035,18 @@ common::timestamp bag_rdr::view::iterator::get_current_msg_stamp() const {
   return rec.to_stamp();
 }
 
+std::optional<common::timestamp> bag_rdr::view::iterator::get_next_msg_stamp() const {
+  if (!assert_print(connection_order.size() > 0)) abort();
```

Comment:
If the current message is the last message of the rosbag and we already got it, then I think that `connection_order` size is 0 in this case. Should we return `std::nullopt` if `connection_order.size() >= 0` for extra safety?

Reply:
We should do the same for `get_current_msg_stamp` but it is out of the scope of this PR